scripts/lib/install_node.sh

```diff
@@ -0,0 +1,21 @@
+# Install Node and necessary frameworks/packages
+
+# Install Node.js on Arch Linux
+install_node() {
+  command -v node >/dev/null 2>&1 || {
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash
+    source ~/.nvm.sh
+    nvm install node
```

Comment:
I usually install the specific version of node like `nvm install 13.8`  and then `nvm use 13.8`. It's better to install the specific version of node to remove any bugs associated with different node versions.

Reply:
Makes sense. I think 12.15.0 would be suitable for this as it's the latest stable version.